### ISSUE TITLE
Mandarin Chinese translation of title template, headings, labels

### ISSUE THREAD

**Issue by cicirello** [cicirello/user-statistician#76]
**Summary**
Add Mandarin Chinese translations of title template, headings, labels, etc.  If you volunteer to work on this one, please also help figure out the correct locale code to use. The list at this link https://www.loc.gov/standards/iso639-2/php/English_list.php only lists zh for "Chinese", but unless I'm mistaken there are multiple Chinese languages, such as seen in this list: https://en.wikipedia.org/wiki/List_of_languages_by_number_of_native_speakers. So it is not clear (at least to me) which zh is intended for.

**Potential Contributors**
Only the [StatConfig.py](https://github.com/cicirello/user-statistician/blob/main/src/StatConfig.py) needs to be modified to implement this. Look for a comment block that has a list of steps necessary to contribute a locale. 

**Please target the dev branch with your PR.**

**Comment by Jibendu007** [comment]
@cicirello  can i be assigned to this?

**Comment by cicirello** [comment]
@Jibendu007 Yes. Do you happen to know if zh is the correct code for this? It appears to be the only one I can find for Chinese, but not clear if it is meant to be for Mandarin or not.

**Comment by Jibendu007** [comment]
According to what I have read from the "Internet" IT SEEMS TO BE zh. Will look into it more while working.

**Comment by berviantoleo** [comment]
Yup, for mandarin, we can use `zh` as the language code.

For Multiple Chinese language, I think we usually use the country code also, because mostly depend on the country itself. Like this article:

https://html-shark.com/HTML/LanguageCodes.htm

I also found the answer about that:

https://stackoverflow.com/a/4894634/6948591

![image](https://user-images.githubusercontent.com/15927349/135881123-8b684414-daee-48fb-9e41-2d6eb6213393.png)

**Comment by Jibendu007** [comment]
okay @cicirello  .

**Comment by cicirello** [comment]
@berviantoleo Thanks. That explanation for the language code is helpful.
